server_admin/topics/identity-broker/first-login-flow.adoc

```diff
@@ -64,3 +64,11 @@ Verify Existing Account By Re-authentication::
   User can also re-authenticate with some different identity provider, which is already linked to his {project_name} account.
   You can also force users to use OTP. Otherwise it's optional and used only if OTP is already set for the user account.
 
+==== Automatically Link Existing First Login Flow
+In order to configure an alternative first login flow in which users are automatically linked without prompt create a new flow with the following two authenticators:
```

Comment:
Is it possible to add small note that it's still possible to use the chain with other authenticators? For example something like:

```
NOTE: The described setup with two authenticators is the simplest one, but it is possible to chain with other 
authenticators according to your needs. For example you can add `Review Profile` authenticator to the beginning if 
you still want end users to confirm their profile informations.
```

What do you think?